packages/cli/src/cli/basemaps/action.export.ts

```diff
@@ -31,12 +25,13 @@ import { primeImageryCache, rgbaToHex } from './tileset.util';
 async function tilesetToConfig(
     name: string,
     projection: Epsg,
-    tag: TileMetadataTag,
+    tag: ConfigTag | string,
```

Comment:
Does this also can be `string` now?

Reply:
Not sure, we could look at forcing it to always be `ConfigTag` rather than any string. that way a string would have to be parsed before being used.